### ISSUE TITLE
Hide alpha versions from npm

### ISSUE THREAD

**Issue by raphinesse** [75lb/local-web-server#24]
IMHO alpha versions shouldn't install when using npm without version specifier.

**Comment by 75lb** [comment]
which version of npm are you on? as install behaviour changed recently regarding prerelease versions.. https://github.com/npm/npm/releases/tag/v3.4.1

i have forcefully set `local-web-server@0.5.23` to be the `latest` version on npm.. so this alpha release will no longer be installed unless specifically requested.

thank you for highlighting this :+1:

**Comment by raphinesse** [comment]
I was on npm 2.7.0. Verified that it installs `local-web-server@0.5.23` now. Thanks!

**Comment by 75lb** [comment]
good.. incidentally, did you have any issues with the alpha release? feedback welcome..

**Comment by raphinesse** [comment]
Yes, when I tried to run the alpha4 I got this error

```
$ ws
module.js:340
    throw err;
    ^

Error: Cannot find module 'ansi-escape-sequences'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:289:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/usr/lib/node_modules/local-web-server/bin/cli.js:6:14)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Function.Module.runMain (module.js:457:10)
```

**Comment by 75lb** [comment]
jeez.. i didn't spot this as npm3 doesn't throw or flag the module as missing.. fixed in 1.0.0-alpha.7, thanks..

**Comment by raphinesse** [comment]
you're welcome
